unlock_indicator.c

```diff
@@ -86,6 +90,43 @@ auth_state_t auth_state;
  * resolution and returns it.
  *
  */
+
+static void check_modifier_keys(void) {
+    /* Get state of Caps and Num lock modifiers, to be displayed in
+     * STATE_AUTH_WRONG state */
```

Comment:
Outdated comment but otherwise LGTM. @Airblader ?

Reply:
Yeah looks good to me, too.